frontend/src/v4/routes/viewerGui/components/measurements/measurements.styles.tsx

```diff
@@ -33,13 +34,21 @@ export const ViewsContainer = styled(ViewerPanel).attrs({
 	min-height: ${VIEWER_PANELS_MIN_HEIGHTS[VIEWER_PANELS.MEASUREMENTS]}px;
 `;
 
-export const Container = styled.div`
+export const ContainerComponent = styled.div`
 	display: flex;
 	flex-direction: column;
 	overflow: auto;
 	flex: auto;
 `;
 
+export const Container = forwardRef(({ children, ...props }: any, ref: RefObject<any>) => (
+	<ViewerScrollArea>
+		<ContainerComponent ref={ref} {...props}>
+			{children}
+		</ContainerComponent>
+	</ViewerScrollArea>
+));
+
```

Comment:
what happens if ContainerComponent is just a ViewerScrollArea ?

Reply:
Also if you have to defnie a component do it in another .tsx file,

Reply:
Container is now a ViewerScrollArea